lib/ddtrace/contrib/rails/action_controller.rb

```diff
@@ -52,6 +52,8 @@ def self.finish_processing(payload)
 
             span.set_tag(Ext::TAG_ROUTE_ACTION, payload.fetch(:action))
             span.set_tag(Ext::TAG_ROUTE_CONTROLLER, payload.fetch(:controller))
+            span.set_tag(Ext::TAG_VIEW_RUNTIME, payload.fetch(:view_runtime))
```

Comment:
Maybe this is a silly question, but where does `view_runtime` come from in the payload? I didn't see this populated when the payload is created, and it's unclear how it gets added.

Reply:
Rails does a little dance to get it added.